safe-room.lic

```diff
@@ -60,6 +60,7 @@ class SafeRoom
         'Come along.  Leave room for the others.',
         "I think you don't really need healing",
         "Now move along, I can't do anything more for you now",
+        "You're in good enough shape, #{Regexp.quote(checkname)}",
```

Comment:
You don't need to Regexp.quote here, you can match directly on #{checkname}

The output of the two is the same.

```
>;e echo DRC.message("#{checkname}")
Mahtra
```
```
 >;e echo DRC.message("#{Regexp.quote(checkname)}")
 Mahtra
 ```

Reply:
Oh, I see you're just sticking with the existing convention already in the file.

Looks good.

Thanks for the contribution!